### PR TITLE
CNV-12542: Adding new feature for Storage RN

### PR DESCRIPTION
**Pull request by bgaydosrh**
for 4.8 only.

Jira: https://issues.redhat.com/browse/CNV-12542

Tagging @yaacov  for code review.
Tagging @duyanyan for QE review.

Direct doc preview link: https://deploy-preview-34302--osdocs.netlify.app/openshift-enterprise/latest/virt/virt-4-8-release-notes?utm_source=github&utm_campaign=bot_dp#virt-4-8-storage-new

### CODE REVIEW

**Review by yaacov — APPROVED** [review]

**Comment by yaacov** [comment]
LGTM

**Comment by gouyang** [comment]
LGTM.

**Comment by duyanyan** [comment]
LGTM

File: virt/virt-4-8-release-notes.adoc [hunk 1]
```diff
@@ -78,6 +78,8 @@ The SVVP Certification applies to:
 
 //CNV-12273 CDI now follows the OpenShift cluster-wide proxy configuration when importing virtual machine disk images.
 
+* You can now prepare a custom disk image, based on an existing disk image, for use as a boot source. Using the {VirtProductName} console, you can customize any template that has a defined source by running a wizard. The wizard steps you through the process of preparing a custom disk image, creating a boot source from the custom disk image, and attaching the boot source to a custom template.
```
Comment (ousleyp) [inline virt/virt-4-8-release-notes.adoc:81]: I might rephrase the second sentence to: `You can customize any template that has a defined source by using a wizard in the {VirtProductName} console.`

**Review by ousleyp — COMMENTED** [review]
One nit. I'd also like to see an xref to the content, if possible. Otherwise, LGTM. Let me know when you're ready for me to merge and approve. Thanks!

File: virt/virt-4-8-release-notes.adoc (hunk 1 above)
Comment (bgaydosrh) [reply virt/virt-4-8-release-notes.adoc]: Done.

**Review by ousleyp — APPROVED** [review]

**Pull request merged**